roles/docker_image_build/vars/main.yml

```diff
@@ -14,6 +14,8 @@ run_commands: "{{ role_run_commands + common_run_commands + host_run_commands }}
 
 build_variants: "{{ role_build_variants + common_build_variants + host_build_variants }}"
 
+docker_labels: "{{ role_docker_labels | combine(common_docker_labels) | combine(host_docker_labels) }}"
```

Comment:
Thoughts on adding some `role_docker_labels`?  Perhaps at least build time, like we do for the [default indexed fields](https://cd.splunkdev.com/sas/ansible/playbooks/ansible_splunk_zero/-/blob/main/environments/docker_dev/group_vars/docker_build_images.yml#L135).